docs/designers-developers/developers/richtext.md

```diff
@@ -56,7 +58,9 @@ registerBlockType( /* ... */, {
 	},
 
 	save( { attributes } ) {
-		return <RichText.Content tagName="h2" value={ attributes.content } />; // Saves <h2>Content added in the editor...</h2> to the database for frontend display
+		const blockProps = useBlockProps.save();
+
+		return <RichText.Content { ...blockProps } tagName="h2" value={ attributes.content } />; // Saves <h2>Content added in the editor...</h2> to the database for frontend display
```

Comment:
Alternatively this could be `<h2 {...blockProps}><RichText.Content value="" /></h2>`, but I don't mind either way.